README.rst

```diff
@@ -433,20 +433,21 @@ Multiline steps
 
 As Gherkin, pytest-bdd supports multiline steps
 (a.k.a. `Doc Strings <https://cucumber.io/docs/gherkin/reference/#doc-strings>`_).
-But in much cleaner and powerful way:
 
 .. code-block:: gherkin
 
     Feature: Multiline steps
         Scenario: Multiline step using sub indentation
             Given I have a step with:
+                """
                 Some
```

Comment:
I just realised, we should also inject the docstring into the step functions (on a separate PR of course) before we release v8.0.0, otherwise it will be another backwards incompatible change